components/Cart/cart.html

```diff
@@ -41,15 +41,15 @@
 					</div>
 					<div class="badge-wrapper"
 						><li class="list-style">
-							<a class="link" href="#">
+							<a class="link" href="/components/Wishlist/wishlist.html">
 								<i class="fa fa-heart nav-icons"></i>
 								<div class="badge icon">5</div>
 							</a>
```

Comment:
![badge](https://user-images.githubusercontent.com/95222845/154898983-f77db06c-e179-4e58-87f7-246aa4614839.png)
The text is barely readable here, check the contrast ratio.
Try using a dark background color if using a light text color and vice-versa.